packages/contracts-bedrock/layout-lock.json

```diff
@@ -0,0 +1,60 @@
+{
+  "GasPriceOracle": {
+    "spacer_1_0_32": {
```

Comment:
This file looks weird because it's only being used for spacers, but slot/offset/length will be important if we add other variables to this file.